CIP-0008/CIP-0010.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
This file is out of place: For traceability could you revert or reset back 1? 
will merge your PR, then add on it #42 after

Reply:
Oh, I see you forced pushed I think. Let me push a valid rebase.